crates/sui-replay-2/Cargo.toml

```diff
@@ -36,8 +36,8 @@ thiserror.workspace = true
 tokio = { workspace = true, features = ["full"] }
 tracing.workspace = true
 
-# [features]
```

Comment:
Not sure why this was commented out but tracing will not work without it (as we can't build with `trace` feature flag without it)

Reply:
I did it just because I was not sure if we compiled always with that on or what.
Sounds good though